app/src/pages/inside/common/launchFiltersToolbar/launchFiltersToolbar.jsx

```diff
@@ -257,57 +285,62 @@ export class LaunchFiltersToolbar extends Component {
           <div className={cx('separator')} />
           <div className={cx('add-filter-button')}>
             <GhostButton
-              icon={AddFilterIcon}
+              icon={this.getFilterIcon(hasFilterPermissions)}
               onClick={this.handleFilterCreate}
-              disabled={!canWorkWithFilters(userRoles)}
+              preventParser={this.state.isFilterCreate}
             >
```

Comment:
Would it be easier to pass icons as children to avoid preventParser?

Reply:
Then it will be necessary to remodel the GhostButton and all the places that use this button.